source/PlayerInfo.h

```diff
@@ -210,6 +210,8 @@ class PlayerInfo {
 	// Check to see if there is any mission to offer right now.
 	Mission *MissionToOffer(Mission::Location location);
 	Mission *BoardingMission(const std::shared_ptr<Ship> &ship);
+	// Last boarding mission offered.
+	bool CaptureOverriden(const std::shared_ptr<Ship> &ship) const;
```

Comment:
This comment doesn't really seems descriptive to me.
```suggestion
	// Return true if the given ship is capturable only because it's the source of a boarding mission which allows it to be.
	bool CaptureOverriden(const std::shared_ptr<Ship> &ship) const;
```